2/contrib/jenkins/kube-slave-common.sh

```diff
@@ -31,6 +31,18 @@ if [[ `grep CentOS /etc/redhat-release` ]]; then
   NODEJS_SLAVE=${NODEJS_SLAVE_IMAGE:-openshift/jenkins-agent-nodejs-8-centos7:${JENKINS_SLAVE_IMAGE_TAG}}
   MAVEN_SLAVE=${MAVEN_SLAVE_IMAGE:-openshift/jenkins-agent-maven-35-centos7:${JENKINS_SLAVE_IMAGE_TAG}}
 fi
+# if the master is running the rhel image, but we are on a pre-3.11 cluster, use the non-TBR images.
+if [[ `grep Enterprise /etc/redhat-release` ]]; then
+  versions=`oc version`
+  echo "OpenShift client and server versions are ${versions}"
+  countForInPodCheck=`echo $versions | awk -F"v3" '{print NF-1}'`
+  # if 2 instance of 'v3', then we are running in a pre3.11 pod;
+  # in 3.11, the server version reporting only lists kubernetes 1.11
```

Comment:
I'm seeing something different, though it may be a client issue:

```
$ oc version
oc v3.11.0+0c1c28f-124
kubernetes v1.11.0+d4cacc0
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://api.ci.openshift.org:443
openshift v3.11.0+84fe0ad-23
kubernetes v1.11.0+d4cacc0
```

so i do see v3 twice in the output.

Reply:
might be better to look for "openshift v3" and then check if it's "openshift v3.11" or not.

Reply:
yeah my tests were with the `oc` in the image from within a jenkins pod ... is that what you did?  

that said, I debated the extra tests like you noted vs. KISS and initially sided with the latter

I'll play with the extra verifications and see what it looks like

Reply:
also, I *think* it looked different from within a pod when I `oc rsh ...` in  vs. what came out of the jenkins start up script 

I'll double check that as well and post the results

Reply:
i built an oc client binary from origin/release-3.11.

Reply:
ok ... so yeah, maybe *eventually* a 3.11 refresh of the rhel image will pick up a later version of `oc` and it will have the openshift v3 line for the server as well.

I just brought up the 3.11 rhel image and `oc` is at `3.11.88`.  And only one `v3.11` line.